src/tribler-core/tribler_core/conftest.py

```diff
@@ -186,7 +186,7 @@ def metadata_store(tmp_path):
 
 @pytest.fixture
 def tags_db():
-    db = TagDatabase()
+    db = TagDatabase(create_tables=True)
```

Comment:
**Question:** Do we really have to pass this keyword argument every time we create the database? Could we instead make a `create_tables` kwarg with default value `True` in the constructor of `TagDatabase`?

Reply:
Yes, we could :) I'll implement it.

Reply:
Done.